coulomb-testkit/src/main/scala/coulomb/arb/ArbQuantity.scala

```diff
@@ -0,0 +1,19 @@
+package coulomb.arb
```

Comment:
I agree it makes sense for this to be a separate library.  I would call the library `coulomb-scalacheck` and the package name `coulomb.scalacheck`